kalamine/cli_msklc.py

```diff
@@ -41,12 +41,19 @@ def cli() -> None: ...
     is_flag=True,
     help="Keep shortcuts at their qwerty location",
 )
+@click.option(
+    "--install",
+    "-i",
+    is_flag=True,
+    help="Install/Update the driver if the creation succeeded",
+)
```

Comment:
It would be more consistent for the user if it was a command rather than an option, as for the XKB counterpart : `xkalamine install layout.toml`.

And for the code architecture, making it a command would favor code composition (e.g. calling the DLL builder first, then the installer builder, then sys-calling the installer), which is easier to test and maintain.